be/src/vec/exec/vempty_set_node.cpp

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
+// t
```

Comment:
Mistaken operation？

Reply:
will remove it. In pr pipeline, if be code is not modified, then pre-compiled binary will be used. but yesterday there was a bug in thirdparty.